questions/src/index.ts

```diff
@@ -33,7 +33,8 @@ const app = express();
 app.use(cors(corsOptions));
 app.use(compression());
 app.use(bodyParser.json());
-app.use("/api/v1/", getFirebaseMiddleware(firebaseAuth), router());
+app.use("/api/v1/", getFirebaseMiddleware(firebaseAuth), normalRouter());
+app.use("/api/serverless", decryptRequestBody(), normalRouter());
```

Comment:
Is this meant to be `serverlessRouter()`?

Reply:
oh yeah. sorry.